From: Jan-Ulrich Holtgrave <uholtgrave@googlemail.com>
Date: Wed, 21 Oct 2020 12:50:19 +0200
Subject: [PATCH] Add Date to RSS entry

---

diff --git a/wp-content/plugins/admin-dashboard-rss-feed/admin-rss-feed.php b/wp-content/plugins/admin-dashboard-rss-feed/admin-rss-feed.php
index 01dd3723..bcab90a0 100644
--- a/wp-content/plugins/admin-dashboard-rss-feed/admin-rss-feed.php
+++ b/wp-content/plugins/admin-dashboard-rss-feed/admin-rss-feed.php
@@ -247,7 +247,8 @@ function wsx_rss_feed_create_box()
 	    foreach($xml->channel->item as $entry) {
 	    	if($in <= $feed_count) {
 	        	echo "<li><a href='$entry->link' title='$entry->title'>" . $entry->title . "</a>";
-	        	echo "<p>" . wp_html_excerpt($entry->description, 120) . "...</p></li>";
+	        	echo "<p style='color:grey'>" . date_i18n("j. F Y", strtotime($entry->pubDate)) . "</p>";
+               echo "<p>" . wp_html_excerpt($entry->description, 150) . "...</p></li>";
 	        	$in++;
 	        }
 	        
